nvflare/apis/study_manager_spec.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+
+class Study:
+
+    def __init__(
+            self,
+            name: str,
+            description: str,
+            sites: [str],
+            users: [str],
+            start_time: datetime.datetime,
+            end_time: datetime.datetime,
+            reviewers=None
+    ):
+        self.name = name
+        self.description = description
+        self.sites = sites
+        self.users = users
+        self.start_time = start_time
+        self.end_time = end_time
+        self.reviewers = reviewers
+        self.create_time = None
+
+
+class StudyManagerSpec(object):
```

Comment:
This should be an abstract class.
And we have a valid implementation somewhere.

But I understand we are doing this now for convenience.